pkg/neorpc/result/block_notifications.go

```diff
@@ -0,0 +1,12 @@
+package result
+
+import (
+	"github.com/nspcc-dev/neo-go/pkg/core/state"
+)
+
+// BlockNotifications represents notifications from a block organized by trigger type.
+type BlockNotifications struct {
+	PrePersistNotifications  []state.ContainedNotificationEvent `json:"prepersist,omitempty"`
```

Comment:
I think we better have `onpersist` instead of `prepersist` in JSON, while `pre` is technically more correct here we have `on` used in trigger, method and many other places, so it'd be a bit more consistent this way.